README.md

```diff
@@ -186,6 +186,36 @@ The format for an unstructured profile is below:
     }
 }
 ```
+
+The format for a graph profile is below:
+```
+"num_nodes": int,
+"num_edges": int,
+"categorical_attributes": list[string],
+"continuous_attributes": list[string],
+"avg_node_degree": float,
+"global_max_component_size": int,
+"continuous_distribution": {
+    "attribute1_name": {
```

Comment:
is it actually `attribute1_name` or is it `<attribute_name>`

Reply:
+1 https://github.com/capitalone/DataProfiler/pull/597/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R326

Reply:
https://github.com/capitalone/DataProfiler/pull/597/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R334